### ISSUE TITLE
[SPELL/SHAMAN] Ancestral Healing

### ISSUE THREAD

**Issue by lingxDEV** [The-Cataclysm-Preservation-Project/TrinityCore#212]
**Current behaviour:**

The effect resets every time it applies.

**Expected behaviour:**

Let's say you have 1000 hp.

You cast a heal for 10 hp.

Now you should have the Ancestral Vigor buff for 1 hp.

Second cast, also a 10 hp heal.

Now you should have the Ancestral Vigor buff for 2 hp. 

So the amount healed will add 10% of that value to the Ancestral Vigor buff up until 10% of the targets max hp.

TL;DR the effect should stack.

**Steps to reproduce the problem:**

1. Make a shaman, spec into Ancestral Healing
2. Cast Riptide, see that every time it heals, the Ancestral Vigor buff resets to 10% of that heal, instead of stacking onto the already existing buff.

**TC rev. hash/commit:** 

5c0febcfb38828a28da9edd27900a85e052cc738

**TDB version:** 

TDB 434.20091

**Operating system:** 

Win10 64

**Comment by lingxDEV** [comment]
Why did you close this @Ovahlord

**Comment by Faq** [comment]
fixed with https://github.com/The-Cataclysm-Preservation-Project/TrinityCore/commit/4a158fdf22089a3fffacc9df19b814c26452e658

**Comment by lingxDEV** [comment]
I'm fucking blind....................................

Good work Ovah
